API/API.cs

```diff
@@ -0,0 +1,53 @@
+using System;
+using System.Collections.Generic;
+using System.Fabric;
+using System.IO;
+using System.Linq;
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.ServiceFabric.Services.Communication.AspNetCore;
+using Microsoft.ServiceFabric.Services.Communication.Runtime;
+using Microsoft.ServiceFabric.Services.Runtime;
+using Microsoft.ServiceFabric.Data;
+
+namespace API
```

Comment:
We should come up with a more descriptive name for this namespace. Naming in general will need some work, but this is minor.

Reply:
Yes I agree. The API isn't really an API. it just calls the Aggregator API to fetch the data and displays it as plain text on a web page. Long term this service isn't needed and data can be ported to SFX or any other place.